auth_proxy/models/user.py

```diff
@@ -16,3 +17,26 @@ class User(Base):
 
     id = Column(Integer, primary_key=True)
     patient_id = Column(String)
+    username = Column(String)
+    password = Column(String)
+    authenticated = Column(Boolean, default=False)
+
+    def is_active(self):
+        """ True, as all users are active.
+        """
+        return True
+
+    def get_id(self):
+        """ Return the user id.
+        """
+        return str(self.id)
+
+    def is_authenticated(self):
```

Comment:
Yeah, it's required by https://flask-login.readthedocs.io/en/latest/#your-user-class. I doesn't need to be stored in the db though, so I'll fix that.